### DISCUSSION TITLE
Transfert PC/Smartphone Androïd

### DISCUSSION THREAD

**Discussion by Esteban49130** [localsend/localsend#1160] [L-Français]
Bonjour, bon transfert Androïd/PC, mais impossible dans l'autre sens PC/Androïd ... erreur 500
Merci pour le dépannage
EB

**Comment by Mench59** [comment]
Salut il faut créer deux règles de pare-feu une en TCP et une en UDP en profil privé pour le trafic entrant et pour le trafic sortant avec le Port Local 53317 : 

En gros ça donne ça :

**Nom: LocalSend_app
Profil: Privé
Action: Autoriser la connexion
Direction: Entrant et sortant
Protocole : TCP & UDP
Port local: 53317**

Salut !

**Comment by Esteban49130** [comment]
Bonjour,Merci de m'en dire un peu plus : créer les règles de pare-feu : où (ordi et/ou androïd, sur l'appli localsend ou ailleurs) ? comment ?  envoyé : 10 mars 2024 à 20:38de : Fabrice ***@***.***>à : localsend/localsend ***@***.***>cc : Esteban49130 ***@***.***>, Author ***@***.***>objet : Re: [localsend/localsend] Transfert PC/Smartphone Androïd (Discussion #1160) Salut il faut créer deux règles de pare-feu une en TCP et une en UDP pour le trafic entrant et pour le trafic sortant avec le port 53317 :En gros ça donne ça :Nom: LocalSend_app Action: Autoriser la connexion Direction: Entrant et sortant Protocole : TCP & UDP Port: 53317Salut !—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

**Reply by Mench59** [reply]
Les règles de pare-feu se font dans le pare-feu Windows Defender (si vous êtes sous windows). Si vous ne savez pas comment créer des règles de pare-feu, je vous invite à suivre un tutoriel en ligne, il y en a plein sur YouTube et plus largement sur la toile.

**Comment by Esteban49130** [comment]
Bonsoir Ça fonctionne ! Merci Envoyé de mon Galaxy model_name Orange
-------- Message d'origine --------De : Fabrice ***@***.***> Date : 11/03/2024  13:57  (GMT+01:00) À : localsend/localsend ***@***.***> Cc : Esteban49130 ***@***.***>, Author ***@***.***> Objet : Re: [localsend/localsend] Transfert PC/Smartphone Androïd (Discussion #1160) 
Les règles de pare-feu se font dans le pare-feu Windows Defender (si vous êtes sous windows). Si vous ne savez pas comment créer des règles de pare-feu, je vous invite à suivre un tutoriel en ligne, il y en a plein sur YouTube et plus largement sur la toile.

**Reply by Mench59** [reply]
Génial, ravi que ça fonctionne. Il est vrai que **LocalSend** bug au niveau de la création des règles de pare-feu, cependant une fois résolu ce logiciel est une merveille. Avant, j'utilisai **Airmore** mais celui-ci utilise le protocole IP et il faut passer par un navigateur sur le pc et l'app sur tel pour faire la liaison PC / Smartphone ce qui me chagrinait légèrement au niveau de la sécu. Bref…
Bonne journée !

**Comment by Esteban49130** [comment]
Oui, en effet le transfert est plus sécure ...Bonne journéeenvoyé : 14 mars 2024 à 11:52de : Fabrice ***@***.***>à : localsend/localsend ***@***.***>cc : Esteban49130 ***@***.***>, Author ***@***.***>objet : Re: [localsend/localsend] Transfert PC/Smartphone Androïd (Discussion #1160) Génial, ravi que ça fonctionne. Il est vrai que LocalSend bug au niveau de la création des règles de pare-feu, cependant une fois résolu ce logiciel est une merveille. Avant, j'utilisai Airmore mais celui-ci utilise le protocole IP et il faut passer par un navigateur sur le pc et l'app sur tel pour faire la liaison PC / Smartphone ce qui me chagrinait légèrement au niveau de la sécu. Bref… Bonne journée !—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
